gui/test/integration/cypress/support/page_files/NodeSearchResultsPage.ts

```diff
@@ -0,0 +1,30 @@
+import {Frame} from "./Frame";
+
+export class NodeSearchResultsPage extends Frame {
+  visit() {
+    cy.visit("/");
+    return this;
+  }
+
+  getVisualizationContainer() {
+    return cy.get("[data-cy=visualizationContainer]");
+  }
+
+  getNodeResultsTable() {
+    return cy.get("[data-cy=matchingNodesTable]");
+  }
+
+  getNodeResultsTableRow(index: number) {
```

Comment:
Even when you need a param to the getter, I usually don't prefix it with get in Cypress page objects, to fit with the nullary get's.